go.mod

```diff
@@ -36,38 +36,39 @@ require (
 	github.com/spf13/cobra v1.6.1
 	github.com/stretchr/testify v1.8.1
 	github.com/xinsnake/databricks-sdk-golang v0.1.3
-	github.com/zalando/go-keyring v0.1.1
-	golang.org/x/crypto v0.4.0
-	golang.org/x/oauth2 v0.3.0
+	github.com/zalando/go-keyring v0.2.2
```

Comment:
Be careful about upgrading this, it has broken things in the past (see `excludes` below)

Reply:
I pinned it to v0.1.1. Thanks!